src/api-reference/financial-integration/v1.financial-integration.markdown

```diff
@@ -0,0 +1,1228 @@
+
+Code|Description|Category
+---|---|---
+0|Successfully processed|Any
+99|System Id in request does not match system id in FI database|Any
+101|This document was previously acknowledged|Acknowledge
+102|This document has been recalled|Acknowledge
+103|This document is not ready|Acknowledge
+104|This document does not exist in FI database|Acknowledge
+105|This document is not of type (expense, invoice, cashadvance, payroll)|Acknowledge
+111|This document has not been acknowledged|Posting
+112|This document has been recalled|Posting
+113|Confirmation has been posted for this document|Posting
+114|Document is not in a known state|Posting
+115|This document is not of type (expense, invoice, cashadvance, payroll)|Posting
+116|This document does not exist in FI|Posting
+198|Invalid request - this same request will not work if tried again.|Posting
+199|Unknown error * please try again later|Any
+
+## <a name="gettransactions"></a>GET a list of Financial Transactions based on a search criteria
+
+Making a GET request to the financial transactions API will return a list of available financial documents, ready to be processed.
+
+### Request
+
+#### Headers
+
+* `OuttaskCompanyId` is a Concur specific custom header for the Outtask Company Id in Concur
+* `entityid` is a Concur specific custom header for the entity code
+
+#### Parameters
+
+Name|Type|Format|Description
+---|---|---|---
+`companyId`|`string`|`path`|**Required** Company Outtask Id in Concur
+`docType`|`string`|`path`|**Required** The financial document type to return. VALUES: expense; invoice; cashadvance. Only one type of transactions can be retrieved at a time
+`offset`|`string`|`query`|Starting page offset
+`limit`|`int32`|`query`|Number of records to return per page (default 100)
+`docId`|`string`|`query`|The transaction unique identifier, it could be expense report ID, payment request ID or Cash Advance ID. If specified, a single document which matches docId is returned
+`ignoreDocumentStatus`|`string`|`query`|Ignores the financial documents status. FORMAT: yes or no. if yes, a document is returned regardless of status. If no, only documents that have not been acknowledged/confirmed are returned
+`systemId`|`string`|`query`|The external system ID that processed the document
+
+##### URI Template
+
+```
+/financialintegration/fi/v1/companies/{companyId}/transactiontypes/{docType}/transactions
+```
+
+#### Payload
+
+None
+
+### Response
+
+#### Payload
+
+[Financial Documents](#schema-fidocuments)
+
+### Example
+
+#### Request
+
+```shell
+GET https://us.api.concursolutions.com/financialintegration/fi/v1/companies/{CompanyID}/transactiontypes/expense/transactions?limit=15
+OuttaskCompanyId: {ID}
+Authorization: Bearer {token}
+```
+#### Response
+
+```shell
+HTTP/1.1 200 OK
+Content-Type: application/json
+Date: date-requested
```

Comment:
This should be `{date-requested}` since you are showing a placeholder (variable) rather than an actual true value for the header.